src/index.tsx

```diff
@@ -1,3 +1,4 @@
+import ModalsContextProvider from 'modal-context/modal-context';
```

Comment:
you are not using this in the file. Check your eslint configs if it works. Also, I think it will be better to move redux provider to App.tsx file